SAP-BAPI-GoodsMovementCreate/workflow.json

```diff
@@ -0,0 +1,135 @@
+        "actions": {
+            "Condition": {
+                "actions": {
+                    "Success_Response": {
+                        "inputs": {
+                            "body": "@body('[RFC]_Call_function_in_SAP')?['content']?['GOODSMVT_HEADRET']",
+                            "headers": {
+                                "Content-Type": "application/json"
+                            },
+                            "statusCode": 200
+                        },
+                        "kind": "Http",
+                        "runAfter": {
+                            "[BAPI]_Commit_transaction": [
+                                "SUCCEEDED"
+                            ]
+                        },
+                        "type": "Response"
+                    },
+                    "[BAPI]_Commit_transaction": {
+                        "inputs": {
+                            "parameters": {
+                                "closeSession": true,
+                                "sessionId": "@body('[BAPI_-_RFC]_Create_stateful_session')?['sessionId']",
+                                "wait": true
+                            },
+                            "serviceProviderConfiguration": {
+                                "connectionName": "sap",
+                                "operationId": "bapiCommit",
+                                "serviceProviderId": "/serviceProviders/sap"
+                            }
+                        },
+                        "type": "ServiceProvider"
+                    }
+                },
+                "else": {
+                    "actions": {
+                        "Failure_Response": {
+                            "inputs": {
+                                "body": "@body('[RFC]_Call_function_in_SAP')?['content']?['RETURN']",
+                                "headers": {
+                                    "Content-Type": "application/json"
+                                },
+                                "statusCode": 400
+                            },
+                            "kind": "Http",
+                            "runAfter": {
+                                "[BAPI]_Roll_back_transaction": [
+                                    "SUCCEEDED"
+                                ]
+                            },
+                            "type": "Response"
+                        },
+                        "[BAPI]_Roll_back_transaction": {
+                            "inputs": {
+                                "parameters": {
+                                    "closeSession": true,
+                                    "sessionId": "@body('[BAPI_-_RFC]_Create_stateful_session')?['sessionId']"
+                                },
+                                "serviceProviderConfiguration": {
+                                    "connectionName": "sap",
```

Comment:
These will need to be changed accordingly as well. I approved the workflow.json to be a part of it, so the test output now suggests changes. For example,

 "referenceName": "sap" => "referenceName": "sap_#workflowname#"